Packet++/header/TcpReassembly.h

```diff
@@ -359,6 +426,8 @@ class TcpReassembly
 	void handleFinOrRst(TcpReassemblyData* tcpReassemblyData, int sideIndex, uint32_t flowKey);
 
 	void closeConnectionInternal(uint32_t flowKey, ConnectionEndReason reason);
+
+	void insertIntoCleanupList(uint32_t flowKey, time_t currentTime = time(NULL));

```

Comment:
do you really need the param `currentTime`? the default value `time(NULL)` is always being used

Reply:
Param has been removed